### PR TITLE
Devices page

### PR DESCRIPTION
**Pull request by tiff**
bildschirmfoto 2015-11-13 um 14 54 40

### CODE REVIEW

File: frontend/scripts/controllers/devices_ctrl.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+angular.module("protonet.platform").controller("DevicesCtrl", function($scope, $state, API, Device) {
+  // @Hinnerk comment this in as soon as you have a json api at /admin/api/devices
+  // Device.startFetcher();
+  // $scope.devices = Device.records;
+  // $scope.$on("$destroy", function() {
+  //   Device.stopFetcher();
+  // });
```
Comment (tiff) [inline frontend/scripts/controllers/devices_ctrl.js]: @hinnerk read the comment :)

File: frontend/scripts/controllers/devices_ctrl.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    path: "/dev/snd/xxx",
+    name: "Logitech Speakers",
+    manufacturer: "Logitech",
+    type: "audio_output"
+  }, {
+    path: "/dev/tty0",
+    name: "Unknown Serial Device",
+    manufacturer: "Unknown",
+    type: "usb_serial"
+  }];
```
Comment (tiff) [inline frontend/scripts/controllers/devices_ctrl.js]: @hinnerk read the comment :)

**Comment by tiff** [comment]
What's the status? @hinnerk

**Comment by hinnerk** [comment]
WIP

**Comment by tiff** [comment]
`Closed with unmerged commits`?

**Comment by hinnerk** [comment]
Yes, precisely. ;)

**Pull request closed without merging**